v_3_6_0/worker_template.go

```diff
@@ -300,32 +356,17 @@ coreos:
       {{ range .Hyperkube.Kubelet.Docker.CommandExtraArgs -}}
       {{ . }} \
       {{ end -}}
-      --address=${DEFAULT_IPV4} \
-      --port={{.Cluster.Kubernetes.Kubelet.Port}} \
       --node-ip=${DEFAULT_IPV4} \
+      --config=/etc/kubernetes/config/kubelet-config.yaml \
       --containerized \
       --enable-server \
       --logtostderr=true \
-      --machine-id-file=/rootfs/etc/machine-id \
       --cadvisor-port=4194 \
       --cloud-provider={{.Cluster.Kubernetes.CloudProvider}} \
-      --healthz-bind-address=${DEFAULT_IPV4} \
-      --healthz-port=10248 \
-      --cluster-dns={{.Cluster.Kubernetes.DNS.IP}} \
-      --cluster-domain={{.Cluster.Kubernetes.Domain}} \
       --network-plugin=cni \
       --register-node=true \
-      --allow-privileged=true \
-      --feature-gates=ExpandPersistentVolumes=true,PodPriority=true,CustomResourceSubresources=true \
       --kubeconfig=/etc/kubernetes/config/kubelet-kubeconfig.yml \
       --node-labels="ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
-      --kube-reserved="cpu=200m,memory=250Mi" \
-      --system-reserved="cpu=150m,memory=250Mi" \
-      --eviction-soft='memory.available<500Mi' \
-      --eviction-hard='memory.available<350Mi' \
-      --eviction-soft-grace-period='memory.available=5s' \
-      --eviction-max-pod-grace-period=60 \
-      --enforce-node-allocatable=pods \
       --v=2"
```

Comment:
So not all settings are moved to config files? Wow...

Reply:
Not all, but component scoped